microsoft-365/security/defender-endpoint/vulnerability.md

```diff
@@ -42,6 +42,7 @@ Method|Return Type|Description
 [Get all vulnerabilities](get-all-vulnerabilities.md)|Vulnerability collection|Retrieves a list of all the vulnerabilities affecting the organization
 [Get vulnerability by Id](get-vulnerability-by-id.md)|Vulnerability|Retrieves vulnerability information by its ID
 [List devices by vulnerability](get-machines-by-vulnerability.md)|MachineRef collection|Retrieve a list of devices that are associated with the vulnerability ID
+[List vulnerabilities by machine and software](get-all-vulnerabilities-by-machines.md)|Retrieves a list of all the vulnerabilities affecting the organization per machine and software.
```

Comment:
```suggestion
[List vulnerabilities by machine and software](get-all-vulnerabilities-by-machines.md)|Vulnerability collection||Retrieves a list of all the vulnerabilities affecting the organization per machine and software
```

Reply:
I fixed it by making it as Vulnerability